### PR TITLE
feat: added login and signup page

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Live preview link: 
https://brand-factory-react.netlify.app/

### CODE REVIEW

**Review by sharmayatendra — APPROVED** [review]
clean code practices followed..good to merge

File: src/pages/SignUpPage/SignUpPage.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React,{useReducer} from 'react'
+import {Nav} from '../../components/Nav/Nav';
```
Comment (dusariudaykumar) [inline src/pages/SignUpPage/SignUpPage.jsx:2]: one suggestion instead of importing nav on every page import it in ```App.js```

**Review by dusariudaykumar — APPROVED** [review]
scrnli_4_5_2022_8-46-34 PM
In the cart and wishlist badge  place values  in the center
- the UI looks good
- the code is clean and clear

File: src/pages/SignUpPage/SignUpPage.jsx (hunk 1 above)
Comment (bishnoimukesh) [reply src/pages/SignUpPage/SignUpPage.jsx:2]: ok i will do this.

**Pull request merged**